### DISCUSSION TITLE
pyenv update

### DISCUSSION THREAD

**Discussion by DaehanHa** [pyenv/pyenv#3008] [Q&A] [answered]
My pyenv version is 2.3.4, but I want to download version 3.11.2 through pyenv install command. However, there is version 3.11-dev in the pyenv install list, but there is no version 3.11.2. There is no response even when I type the pyenv update command.

help me plz

![image](https://github.com/pyenv/pyenv/assets/113526396/fd185576-34f1-4f81-b2e0-48ac03962185)
This was when I typed pyenv install -list.
<img width="439" alt="pyenv_version" src="https://github.com/pyenv/pyenv/assets/113526396/d9ecb26d-85c9-413a-85ef-cbc7261fbee5">
This was when I typed pyenv --version.

**Comment by native-api** [comment, marked as answer]
https://github.com/pyenv/pyenv/wiki/Common-build-problems#python-build-definition-not-found-or-another-new-feature-missing-even-though-you-have-a-new-enough-pyenv
